R/S3.R

```diff
@@ -1805,3 +1805,31 @@ pander.nls <- function(x, digits = panderOptions('digits'), show.convergence = F
     pander.summary.nls(summary(x), summary = FALSE, add.significance.stars = FALSE,
                        digits = digits, show.convergence = show.convergence, ...)
 }
+
+#' Prints an arima object from stats package in Pandoc's markdown.
+#' @param x an arima object
+#' @param digits number of digits of precision
+#' @param se if to include standard error in coefficients table (default \code{TRUE})
+#' @param ... optional parameters passed to raw \code{pandoc.table} function
+#' @export
+pander.Arima <- function(x, digits = panderOptions('digits'), se = TRUE,...) {
+    cat('\nCall:', pandoc.formula.return(x$call), '', sep = '\n')
+    cn <- names(x$coef)
+    coef <- matrix(x$coef, nrow = 1)
+    colnames(coef) <- cn
+    if (se && nrow(x$var.coef)) {
```

Comment:
`var.coef` is the estimated variance matrix of the coefficients, and it's size will always be number of coefficients x number of coefficients. What is being print here is standard error that is calculated from diagonal of that matrix, so it will always have 1 row.